### PR TITLE
Fix  #1 and add post-type selector

### LINKED ISSUE #1: Affects other query loop
Very cool although when I try it any subsequent standard query loop blocks on the page seem to pick up the post__in parameter and therefore don't return the expected data.

### PR DESCRIPTION
**Pull request by jtsternberg**
This PR addresses issue #1, and it also adds a post-type selector so you can search/select/display any post-type.

### CODE REVIEW

**Comment by coreyworrell** [comment]
Hope this can be merged and released soon!

**Comment by creative-andrew** [comment]
@jtsternberg @coreyworrell,
Excellent work. I will check this and create a release ASAP.

File: single-post-query-loop-selector.php [hunk 1]
```diff
@@ -14,59 +14,103 @@
...
+				$query['post__in'] = $this->parsed_block['attrs']['query']['include'];
+			}
+
+			if ( isset( $this->parsed_block['attrs']['query']['postType'] ) ) {
+				$query['post_type'] = $this->parsed_block['attrs']['query']['postType'];
+			}
+		}
+
+		// Our work here is done (backs away slowly).
+		remove_filter( 'query_loop_block_query_vars', [ $this, 'modify_query_vars' ], 10, 3 );
```
Comment (creative-andrew) [inline single-post-query-loop-selector.php:110]: @jtsternberg 
Thanks for the fixes.

What I don't understand is why we need to remove the filter.

I used Ryan Welcher's example: https://github.com/ryanwelcher/advanced-query-loop/blob/trunk/includes/query-loop.php#L61

Any subsequent query shouldn't be affected if we just add the the namespace check 
`'creativeandrew/single-post-query-loop-selector' === $this->parsed_block['attrs']['namespace']`

But there isn't a reason why to remove the filter per se right?

**Review by creative-andrew — APPROVED** [review]

File: single-post-query-loop-selector.php (hunk 1 above)
Comment (jtsternberg) [reply single-post-query-loop-selector.php:110]: I've had to move on from this project, but I think you're probably right.

**Comment by creative-andrew** [comment]
Closed in favour of https://github.com/creative-andrew/single-post-query-loop-selector/pull/4

**Pull request closed without merging**